UnitTests/BraintreeCoreTests/BTAppContextSwitcher_Tests.swift

```diff
@@ -80,13 +80,19 @@ class BTAppContextSwitcher_Tests: XCTestCase {
         XCTAssertFalse(handled)
         XCTAssertNil(MockAppContextSwitchClient.lastHandleReturnURL)
     }
-        
+
     func testHandleOpenURLContext_withNoAppSwitching_returnsFalse() {
         let mockURLContext = BTMockOpenURLContext(url: URL(string: "fake://url")!).mock
         let handled = BTAppContextSwitcher.sharedInstance.handleOpenURL(context: mockURLContext)
         XCTAssertFalse(handled)
     }
 
+    // MARK: - universalLink Tests
+
+    func testSetUniversalLink() {
+        BTAppContextSwitcher.sharedInstance.universalLink = "https://fake.com"
+        XCTAssertEqual(appSwitch.universalLink, "https://fake.com")
+    }
```

Comment:
This is a similar test that exists for the return URL - there is not a ton we can test until we add the protocol conformance to the client. It seems like most of the return URL tests exist on the client. Do we want any other tests in this file?